src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -106,7 +107,11 @@ def get_approx_start_time(group: Group):
     ),
     "State": lambda group: SUBSTATUS_TO_STR.get(group.substatus, "").replace("_", " ").title(),
     "First Seen": lambda group: time_since(group.first_seen),
-    "Approx. Start Time": get_approx_start_time,
+    "Approx. Start Time": lambda group: datetime.fromtimestamp(
+        get_approx_start_time(group=group)
+    ).strftime(
+        "%Y-%m-%d %H:%M:%S"
+    ),  # format moment into YYYY-mm-dd h:m:s
```

Comment:
i just noticed this, i don't think you need to do this because i have a function here that does it https://github.com/getsentry/sentry/blob/9dbd4671196d2b9aa5dc2e9590d477dd47bf7409/src/sentry/integrations/slack/message_builder/issues.py#L79-L97

Reply:
this also pulls the occurrence and gets an accurate start time